rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -627,7 +627,9 @@ def get_history_events(
             changes = self._process_trove_events(trove['changes'], from_timestamp, to_timestamp)
             result.append(changes)
         # Flatten the result (list of lists to list)
-        return reduce(add, result)
+        if result:
```

Comment:
Thank you very much for finding this issue. Would you mind to change this condition to `len(result) == 0`? At rotki we prefer to be as explicit as possible to avoid possible errors in the future

Reply:
sure, already done :)